wptagent.py

```diff
@@ -490,6 +489,7 @@ def startup(self, detected_browsers):
 
         # Optional imports
         self.requires('fontTools', 'fonttools')
+        self.requires('tzlocal')
```

Comment:
https://github.com/catchpoint/WebPageTest.agent/pull/638/files#diff-292d079fe725eed06ead69e3675d638d8bda0c4b65aae36ebde665c7ed52a2beR20

it introduced a dependency on pytz too
```suggestion
        self.requires('tzlocal')
        self.requires('pytz')
```